build.gradle.kts

```diff
@@ -35,14 +29,14 @@ version = if (!project.hasProperty("useSnapshot") &&
 ) {
     val prefix = when (currentBranch) {
         null, "", "v1.x", "HEAD", "master", "main" -> ""
-        else -> "$currentBranch."
+        else -> "${libs.versions.baseVersion.get()}."
```

Comment:
This looks like a mistake to me

Reply:
No, I mean the code used to put the current branch here before and add the base version two lines down, now it adds the base version twice.

Reply:
You are right, the version catalog is intended for dependency versions and not our own version. I will revert this part.